src/i18n/i18n.json

```diff
@@ -911,5 +912,26 @@
     "theme": "Theme",
     "theme.light": "Light",
     "theme.dark": "Dark"
+  },
+  "userAlerts": {
```

Comment:
All i18n keys are in the singular form - it should be `userAlert`.
Then a search / replace with `i18n.userAlerts` to `i18n.userAlert`.